azure-iot-device/azure/iot/device/iothub/aio/async_clients.py

```diff
@@ -52,8 +52,8 @@ def __init__(self, **kwargs):
 
         :param iothub_pipeline: The IoTHubPipeline used for the client
         :type iothub_pipeline: :class:`azure.iot.device.iothub.pipeline.IoTHubPipeline`
-        :param edge_pipeline: The EdgePipeline used for the client
-        :type edge_pipeline: :class:`azure.iot.device.iothub.pipeline.EdgePipeline`
+        :param edge_pipeline: The MethodInvokePipeline used for the client
```

Comment:
>edge_pipeline [](start = 15, length = 13)

My $.02: I really dislike the name "edge_pipeline"

Reply:
this is fair, I'm changing everything to http_pipeline.

---
In reply to: [352789079](https://github.com/Azure/azure-iot-sdk-python/pull/403#discussion_r352789079) [](ancestors = 352789079)